claat/cmd/export_test.go

```diff
@@ -41,7 +42,6 @@ func TestExportCodelabMemory(t *testing.T) {
 		{
 			name:     "Multiple Steps",
 			filePath: "testdata/simple-2-steps.md",
-			knownBug: "https://github.com/googlecodelabs/tools/issues/391",
```

Comment:
Can we remove the test.knownBug check below now?

Reply:
Removed.